itasks_server/.gitignore

```diff
@@ -1,2 +0,0 @@
-*
```

Comment:
This file is needed to keep this directory in the Git repository

Reply:
readded